Services/Dashboard/ItemsBlock/classes/class.ilPDSelectedItemsBlockGUI.php

```diff
@@ -172,22 +175,23 @@ public function removeFromDeskObject()
     }
 
     /**
-     * @inheritdoc
+     * @inheritDoc
      */
-    public function getBlockType() : string
+    public function getBlockType(): string
     {
         return static::$block_type;
     }
 
     /**
-     * @inheritdoc
+     * @inheritDoc
      */
     public static function getScreenMode()
     {
         $cmd = $_GET['cmd'];
-        if ($cmd == 'post') {
+        if ($cmd === 'post') {
             $cmd = $_POST['cmd'];
-            $cmd = array_shift(array_keys($cmd));
+            $keys = array_keys($cmd);
+            $cmd = array_shift($keys);
```

Comment:
This could be simplified with `$cmd = key($cmd)` or `$cmd = key($_POST['cmd'])`.